website/docs/r/cloud_provider_access.markdown

```diff
@@ -65,4 +75,90 @@ The Cloud Provider Access resource can be imported using project ID and the prov
 $ terraform import mongodbatlas_cloud_provider_access.my_role 1112222b3bf99403840e8934-AWS-5fc17d476f7a33224f5b224e
 ```
 
-See [MongoDB Atlas API](https://docs.atlas.mongodb.com/reference/api/cloud-provider-access-create-one-role/) Documentation for more information.
\ No newline at end of file
+## mongodbatlas_cloud_provider_access (optional)
+
+This is the first resource in the multi-resource path
```

Comment:
Let's call it two-resource, edit to:
This is the first resource in the two-resource path as described above.